openedx/features/student_certificates/helpers.py

```diff
@@ -213,13 +213,13 @@ def get_pdfkit_html(image_base64):
     return PDFKIT_HTML_STRING.format(image_tag=PDFKIT_IMAGE_TAG.format(base64_img=image_base64))
 
 
-def override_update_certificate_context(request, context, course, user_certificate):
+def override_update_certificate_context(request, context, course, user_certificate, preview_mode=None):
     """
     This method adds custom context to the certificate
     :return: Updated context
     """
     border = request.GET.get('border', None)
-    if border and border == 'hide':
+    if (border and border == 'hide') or preview_mode:
```

Comment:
can we encapsulate conditionals here by writing a small method in helpers.py
```
def _should_hide_border(border, preview_mode):
    return (border and border == 'hide') or preview_mode
```

and update context 
```
context['border_class'] = 'certificate-border-hide' if _should_hide_border(border, preview_mode) else ''
```

Reply:
We can also simplify this line `(border and border == 'hide') or preview_mode` with 
`border == 'hide' or preview_mode`
as if border is None then `None == 'hide'` will return False, means it will behave in the same way.